README.md

```diff
@@ -1,40 +1,41 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Co-Design Project
 
-## Getting Started
-
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
-```
-
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+## Table of Content
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+- [Introduction](#introduction)
+- [Get Started](#getting-started)
+- [Prerequisites](#prerequisites)
+- [Deployment](#deployment)
```

Comment:
Here before get started you can also add a title for the technology used like:

![Screenshot Capture - 2024-06-18 - 10-19-59](https://github.com/EnAccess/co-design-FE/assets/77101006/11832ba4-89ba-45e2-95f0-fb392d8ea2f5)

Reply:
Fixed